.azure/pipelines/components-e2e-tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter Traits\!~QuarantinedTest --logger trx
```

Comment:
Suggest doing this step twice, once each for quarantined and unquarantined tests. Also add another `PublishTestResults@2` for TestResults/Quarantined, similar to https://github.com/dotnet/aspnetcore/blob/d4d4c4bda5f36a8ecb7cc5c66ef4ad6ae72f1df5/.azure/pipelines/quarantined-pr.yml#L94-L102

Reply:
Note: Without this "suggestion", quarantined component tests won't run anywhere

Reply:
Thanks updated via https://github.com/dotnet/aspnetcore/pull/38693/commits/05b84caf7fcb8e17e039a453f1e11ac22877e91f. However not sure if the publish step will really work, as the test results would just go to the `TestResults` directory (I don't see what would be creating the actual `Quarantined` directory in this case).

Reply:
>  I don't see what would be creating the actual Quarantined directory in this case

See https://github.com/dotnet/aspnetcore/blob/main/eng/QuarantinedTests.AfterArcade.props

Reply:
Added `-p:RunQuarantinedTests=true` to satisfy the condition.